EIPS/eip-1679.md

```diff
@@ -40,6 +40,7 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 - [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
 - [EIP-1803](https://eips.ethereum.org/EIPS/eip-1803): Rename opcodes for clarity
 - [EIP-1884](https://eips.ethereum.org/EIPS/eip-1884): Repricing for trie-size-dependent opcodes
+- [EIP-1930](https://github.com/ethereum/EIPs/issues/1930): Strict Gas CALL
```

Comment:
I think we should not merge any non-draft EIP into the meta. This particular one must be merged first and have a https://eips.ethereum.org/EIPS/eip-1930 URL.

Reply:
I have requested the 1930 PR to be merged but it is still not merged. I can't do much more about it. On that note, who is responsible for merging PR ?

As for refusing draft EIP, I think this will increase even further the selection bias already present. 

One reason this EIP is in draft is that there could be some discussion around it. But I could also move it to final if that was the only reason blocking it. 

To me the point of EIP-1679 is to discuss about the improvement we want to see implemented sooner rather than latter. 

EIP-1930 is an important improvement for meta-transaction and it would beneficial to have it included in the next hardfork.

Reply:
My reasoning is that an EIP merged as a Draft signals that it has passed the basic formatting tests and has a discussion URL. 

You are linking to an issue -- a discussion URL -- and not an EIP. The meta should point to EIPs and not discussions.

Reply:
OK, I linked to the pull request instead. I actually did not realised it was the link to the issue.